### ISSUE TITLE
Cannot finish local build

### ISSUE THREAD

**Issue by ispmarin** [caddy-dns/porkbun#2]
I've been trying to build a caddy docker image with porkbun DNS using this Dockerfile:

```
FROM caddy:builder AS builder

RUN xcaddy build --with github.com/caddy-dns/porkbun

FROM caddy:latest

COPY --from=builder /usr/bin/caddy /usr/bin/caddy
```

And every single time the build freezes on the go download phase and does not advance:

![image](https://user-images.githubusercontent.com/12247/218164879-dd223fa5-d45b-46d2-b125-d3ffd8d7e719.png)

What could be going on?

Thanks

**Comment by sporez** [comment]
Similar issue here. Building with xcaddy and the process hangs here:

<img width="975" alt="image" src="https://user-images.githubusercontent.com/11649198/218242880-a3af9b98-df22-41eb-8c7b-94dbd02c7876.png">

**Comment by mholt** [comment]
You can work around this by adding `--with github.com/caddyserver/caddy/v2=github.com/caddyserver/caddy/v2@v2.6.3` to your build command... not sure why this fixes it.

This might be related to https://github.com/caddyserver/caddy/issues/5301, but in those cases the `go get` command was panicking, not just hanging.

Since the workaround is the same for both, I wonder if it's the same bug.

The Go team is aware of the bug and knows what the cause is. However due to low morale at Google right now, I'm told it's hard to find motivation to dig in and fix it. Understandably, we will need to wait for a proper fix.

Another solution for porkbun specifically is apparently to update its caddy/v2 dependency to v2.6.3 (the latest release as of now).

**Comment by Niallfitzy1** [comment]
> Another solution for porkbun specifically is apparently to update its caddy/v2 dependency to v2.6.3

The dependency bump works for now at least.

Fixed in #3 with v0.1.1 published.
